### ISSUE TITLE
Additional sync methods?

### ISSUE THREAD

**Issue by emmaexe** [SkyD666/Rays-Android#6]
I like the app. Are there any plans to add support for other sync methods e.g. syncing to a local folder on the device, for synchronization using a service like [syncthing](https://syncthing.net/)? I don't have a WebDAV server available.

**Comment by SkyD666** [comment]
Thank you for your suggestion. New methods will be added in the future.

**Comment by SkyD666** [comment]
This version supports importing and exporting data via files, but it is currently in the testing phase and may have bugs.

APK file: https://github.com/SkyD666/Rays-Android/suites/16862145514/artifacts/963129549

CI: https://github.com/SkyD666/Rays-Android/actions/runs/6404475082

**Comment by SkyD666** [comment]
This version supports importing and exporting data via files, but it is currently in the testing phase and may have bugs

CI: https://github.com/SkyD666/Rays-Android/actions/runs/6414975531

**Comment by SkyD666** [comment]
https://github.com/SkyD666/Rays-Android/releases/tag/32

**Comment by emmaexe** [comment]
> https://github.com/SkyD666/Rays-Android/releases/tag/32

Hi. Sorry for a late reply. I tried the beta and the version 32, and have some thoughts.

In the beta, the exporting/importing works but is not optimal for my original idea of using the syncthing service, since it does not happen automatically after changes are made (sitckers added/modified/deleted) and the export is a zip file, not just a folder containing the BackupData and BackupSticker folders which means after every export instead of syncing changed files from a folder, the entire library of stickers (zip file) has to be re-synced.

As for the version 32, I installed it, but I don't see the option for exporting to the filesystem?

**Comment by SkyD666** [comment]
yeah, it does not happen automatically after changes are made. This will be a future feature. The zip file contains stickers.

**Comment by SkyD666** [comment]
You can export stickers by touching
 home screen "export" or Multi-select stickers which will be exported in the search screen. Sorry for my poor English.
